vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -0,0 +1,77 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.view.RedirectView;
+import edu.asu.diging.vspace.core.data.ExhibitionRepository;
+import edu.asu.diging.vspace.core.data.SpaceRepository;
+import edu.asu.diging.vspace.core.factory.impl.ExhibitionFactory;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.impl.Exhibition;
+import edu.asu.diging.vspace.core.services.impl.ExhibitionManager;
+import edu.asu.diging.vspace.core.services.impl.SpaceManager;
+
+@Controller
+public class ExhibitionConfigurationController {
+
+    @Autowired
+    private SpaceRepository spaceRepo;
+
+    @Autowired
+    private SpaceManager spaceManager;
+
+    @Autowired
+    private ExhibitionManager exhibitManager;
+
+    @Autowired
+    private ExhibitionRepository exhibitRepo;
+
+    @Autowired
+    private ExhibitionFactory exhibitFactory;
+
+    @RequestMapping("/staff/exhibit/config")
+    public String showExhibitions(Model model) {
+        model.addAttribute("exhibitionsList", exhibitRepo.findAll());
```

Comment:
let's add a find all method to the manager, so that this controller isn't dependent on both manager and repository.